curriculum/challenges/english/25-front-end-development/lecture-understanding-performance-in-web-applications/67d2f7e9d895bf7dcef00408.md

```diff
@@ -2,13 +2,51 @@
 id: 67d2f7e9d895bf7dcef00408
 title: What Are Some Techniques for Improving CSS Performance?
 challengeType: 11
-videoId: nVAaxZ34khk
+videoId: LmcHZXY1NLw
 dashedName: what-are-some-techniques-for-improving-css-performance
 ---
 
 # --description--
 
-Watch the lecture video and answer the questions below.
+Watch the video or read the transcript and answer the questions below.
+
+# --transcript--
+
```

Comment:
This is the only video that does not starts with the title.

Reply:
Yea, I think that's fine for now. I did notice a few others in earlier (maybe HTML) lecture. We could create an issue, but I'm not sure it's worth the time to edit.